tests/functional/cli/test_cli_artifacts.py

```diff
@@ -27,7 +27,9 @@
 def test_cli_artifacts(capsysbinary, gitlab_config, gitlab_runner, project):
     project.files.create(data)
 
-    while not (jobs := project.jobs.list(scope="success")):
+    jobs = None
+    while not jobs:
+        jobs = project.jobs.list(scope="success")
```

Comment:
Was this change meant to be part of this PR or for a follow-up?

Also maybe we can just get rid of the skip marker then if your idea was to support < 3.8.

Reply:
It was the only thing preventing me from running the full test suite in my debian VM, figured it was insignificant enough to include it in this PR, but I can create a separate one for this if preferred?

Reply:
No it's ok, I guess I was a little too excited when adding that and 3.8 was still new lol. But we can just remove the skip

Reply:
Done.